salt/utils/gitfs.py

```diff
@@ -1074,7 +1135,9 @@ def gen_lock(self, lock_type="update", timeout=0, poll_interval=0.5):
                         time.sleep(poll_interval)
                         continue
         finally:
-            if lock_set:
+            if lock_set1 or lock_set2:
+                msg = f"Attempting to remove '{lock_type}' lock for '{self.role}' remote '{self.id}' due to lock_set1 '{lock_set1}' or lock_set2 '{lock_set2}'"
```

Comment:
Wrap these long messages please:
```suggestion
                msg = (
                    f"Attempting to remove '{lock_type}' lock for '{self.role}' remote "
                    f"'{self.id}' due to lock_set1 '{lock_set1}' or lock_set2 '{lock_set2}'"
                )
```
Something like these, as long as it's less than 120 chars.

Reply:
done